mungegithub/submit-queue/www/index.html

```diff
@@ -34,7 +34,7 @@ <h2 class="md-toolbar-tools">E2E Tests Failing. Entire submit queue blocked. Las
           <md-tab label="PRs" md-on-select="cntl.selectTab('prs')">
             <md-content class="md-whiteframe-z2">
               <md-toolbar>
-                <h2 class="md-toolbar-tools">Pull Requests</h2>
+                <h2 class="md-toolbar-tools">Pull Requests ({{cntl.prsCount}})</h2>
```

Comment:
Could update that count also [here](https://github.com/grodrigues3/contrib/blob/2c918d7e60403f7a14bf28521adf500f78376289/mungegithub/submit-queue/www/script.js#L354) to give us the length of the returned list when we filter.